game/Bird.java

```diff
@@ -124,11 +124,11 @@ public void update() {
         if (Resources.IN_TUBE) {
             int leftBird = Resources.BIRD_X_POSITION;
             int rightBird = Resources.BIRD_X_POSITION + Resources.BIRD_WIDTH;
-            int leftTube = (int) Resources.CURRENT_TUBE.getX();
-            int rightTube = (int) Resources.CURRENT_TUBE.getX() + Resources.TUBE_WIDTH;
+            int leftTube = (int) Resources.CURRENT_TUBE.getX()+50;
```

Comment:
I have difficulties compiling it, because it's confusing to me (I don't use java) I don't really care that the constants exist, but they should be given names (like pi instead of 3.14159265 - even though that isn't used here) because that helps people who are new to the project understand the code.
(I do agree however, that birds shouldn't collide with thin air)